src/templates/components/ManualCard.svelte

```diff
@@ -0,0 +1,165 @@
+<script context="module" lang="ts">
+	import { CLICK_MACRO } from '$lib/gam';
+</script>
+
+<script lang="ts">
+	import '$templates/components/fonts/Egyptian.css';
+	import '$templates/components/fonts/EgyptianWeb.css';
+	import '$templates/components/fonts/Sans.css';
+	import Button from './Button.svelte';
+	import ArrowRight from './icons/ArrowRight.svelte';
+
+	export let TotalCardNumber: number;
+	export let CardIndex: number;
+	export let EventTitle: GAMVariable;
+	export let EventDateTime: GAMVariable;
+	export let EventImage: GAMVariable;
+	export let EventUrl: GAMVariable;
+	export let direction = 'row';
+
+	let [boldTitle, regularTitle] = EventTitle.split(':');
+	if (regularTitle) {
+		boldTitle += ':';
+	} else {
+		regularTitle = '';
+	}
+</script>
+
+<a
+	class="card-{CardIndex} card split-into-{TotalCardNumber}"
+	href={EventUrl}
+	style={`--direction: ${direction}`}
+>
+	<div class="media">
+		<picture>
+			<img src={EventImage} alt="" />
+		</picture>
+	</div>
+	<div class="text">
+		<h2><b>{boldTitle}</b>{regularTitle}</h2>
+		<p>{EventDateTime}</p>
+	</div>
+	<a class="button" href={`${CLICK_MACRO}${EventUrl}`} target="_top">
+		Book tickets
+		<ArrowRight width={24} />
+	</a>
+</a>
+
+<style>
+	a {
+		color: #000000;
+		text-decoration: none;
+	}
+
+	a.card-1,
+	a.card-2 {
+		padding: 12px 10px;
+		display: block;
+		margin: 0px;
+		width: 50%;
+	}
+
+	a.card-3,
+	a.card-4 {
+		display: none;
+	}
```

Comment:
Would this not be better handled by an [`:nth-of-type` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type)?

Reply:
Yeah nice idea! Implemented 🙂